WalletWasabi.Gui/Config.cs

```diff
@@ -213,43 +199,60 @@ public IPEndPoint GetTorSocks5EndPoint()
 			return _torSocks5EndPoint;
 		}
 
+		public void SetEndpoint(string host, int? port)
+		{
+			if (Network == Network.Main)
+			{
+				MainNetBitcoinCoreHost = host;
+				MainNetBitcoinCorePort = port ?? Network.Main.DefaultPort;
+			}
+			else if (Network == Network.TestNet)
+			{
+				TestNetBitcoinCoreHost = host;
+				TestNetBitcoinCorePort = port ?? Network.TestNet.DefaultPort;
+			}
+			else if (Network == Network.RegTest)
+			{
+				RegTestBitcoinCoreHost = host;
+				RegTestBitcoinCorePort = port ?? Network.RegTest.DefaultPort;
+			}
+			else
+			{
+				throw new NotSupportedException($"Unsupported Network");
+			}
+		}
+		public (string host, int port) GetEndpoint()
```

Comment:
I think this will need to be a real type, especially with https://github.com/zkSNACKs/WalletWasabi/pull/1704